### PR TITLE
old parameter name in javadoc

### PR DESCRIPTION
**Pull request by sevenmaster**

### CODE REVIEW

**Comment by larsk21** [comment]
Isn't the javadoc description wrong as well? Since it's not the number of hours anymore, but a TimeSpan.

**Comment by larsk21** [comment]
The description of the getter seems to be wrong as well. Maybe we should add an issue to check for missing/wrong documentation throughout the project (in addition to the documentation issues we already have). @gstuer Thoughts?

File: src/main/java/data/Profession.java [hunk 1]
```diff
@@ -9,12 +9,12 @@
...
     private final TimeSpan maxWorkingTime;
     private final double wage;
-    
+
     /**
      * Constructs a new {@link Profession profession} instance.
      * @param departmentName - Name of the department this profession is associated with.
      * @param workingArea - {@link WorkingArea} of this profession.
-     * @param maxWorkingHours - Maximum number of hours an {@link Employee employee} may work per month.
+     * @param maxWorkingTime - Maximum number of hours an {@link Employee employee} may work per month.
```
Comment (gstuer) [inline src/main/java/data/Profession.java]: I agree with @larsk21 that the description should also be changed in order to avoid another PR concerning the same documentation issue.

**Comment by larsk21** [comment]
Doesn't seem to be worked on.
